migration-engine/connectors/sql-migration-connector/src/flavour.rs

```diff
@@ -1,5 +1,3 @@
-#![deny(missing_docs)]
```

Comment:
?

Reply:
It's at the top-level in the crate now, so the module-level deny is a noop :)